buildSrc/src/main/groovy/org/elasticsearch/gradle/test/ClusterConfiguration.groovy

```diff
@@ -125,6 +125,8 @@ class ClusterConfiguration {
 
     Map<String, Object> settings = new HashMap<>()
 
+    Map<String, String> keyStoreSetting = new HashMap<>()
```

Comment:
plural please? Also, just for consistency with the code in ES, can we use all lower case for keystore, so `keystoreSettings`?